test/e2e/storage/persistent_volumes-local.go

```diff
@@ -227,7 +227,7 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 
 				BeforeEach(func() {
 					By("Creating pod1")
-					pod1, pod1Err = createLocalPod(config, testVol)
+					pod1, pod1Err = createLocalPod(config, testVol, framework.PodStartShortTimeout)
```

Comment:
Since all calls of `createLocalPod` are using the same timeout value, it could just be used as a constant inside.

Reply:
@msau42 Done and squashed, see https://github.com/kubernetes/kubernetes/pull/62635/files.